### DISCUSSION TITLE
SQL Server database table and fields with space

### DISCUSSION THREAD

**Discussion by Simond-SA** [prisma/prisma#12489] [Q&A]
Hello,
I have an mssql database (from Navision) which has tables and fields names with special spaces and characters such as:

- table name: Sales Invoice Header
- field: No_

I can't make the schematic with Prisma; I have the following error:

Error occurred during query execution:
_ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(TokenError { code: 156, state: 1, class: 15, message: "Incorrect syntax near the keyword 'ASC'."_

In addition, this database is read-only.

Do you have any suggestions for my problem?

**Comment by austincrim** [comment]
Hey @Simond-SA 👋 ,

Does this error occur when running `prisma db pull`? It should automatically map the table names to something that is usable my Prisma.

If it is erroring during this command, a [bug report](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with more details would be great!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
